internal/layer2/announcer.go

```diff
@@ -138,11 +138,15 @@ func (a *Announce) updateInterfaces() {
 func (a *Announce) spamLoop() {
 	// Map IP to spam stop time.
 	m := map[string]time.Time{}
-	// See https://github.com/metallb/metallb/issues/172 for the 1100 choice.
-	ticker := time.NewTicker(1100 * time.Millisecond)
+	ticker := time.NewTicker(time.Hour)
```

Comment:
I know this is a nit-pick, but I'd suggest changing this line back.  It made me wonder what the significance of an hour was, but I'm pretty sure you just picked a "big" number since it was going to be stopped right away.  I'd either add a comment clarifying that, or just change it back to the original value used elsewhere.

Reply:
Added a comment, tell me if it's better

Reply:
that works